algorithms/integration/integrator.py

```diff
@@ -356,6 +362,7 @@ def __init__(self):
         def __init__(self):
             self.fitting = True
             self.sigma_b_multiplier = 2.0
+            self.valid_foreground_threshold = 0.6
```

Comment:
Should read default value from `master_phil`?

Reply:
Unfortunately this hoop jumping is needed due to the way the parameters are constructed in the integration code: `self.params = Parameters.from_phil(params.integration)`. I don't know why it needs to be this way but it won't work without it .....

Reply:
🧐

OK, propose we live with this for now and then add an issue to resolve why this is the case? Because this _will_ end badly at some point in the future.